sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/CosmosAsyncClient.java

```diff
@@ -464,6 +466,16 @@ TracerProvider getTracerProvider(){
         return this.tracerProvider;
     }
 
+    /**
+     * Enable throughput control group.
+     *
+     * @param group Throughput control group going to be enabled.
+     */
+    public void enableThroughputControlGroup(ThroughputControlGroup group) {
```

Comment:
beta API tag please.

Reply:
Do not this enableThroughputControlGroup should be public facing, since we enable through the syncContainer/asyncContainer. 

Changed to hide it now